files/js/app/config/environment.js

```diff
@@ -1,3 +1,22 @@
-import loadConfigFromMeta from '@embroider/config-meta-loader';
+// import loadConfigFromMeta from '@embroider/config-meta-loader';
```

Comment:
either this, or ember-cli, need to be made aware of the possibility of `.cjs` extensions for config/environment